### ISSUE TITLE
Training accuracy

### ISSUE THREAD

**Issue by Mandy-77** [NVlabs/A-ViT#3]
Thanks for your interesting and excellent work.
I rerun the training codes using avit-tiny but only get 68.26% top-1 accuracy on ImageNet, would different training processes cause that much difference?
Additionally, how to actually 'remove' those stopped tokens in the inference stage to reduce inference time when batchsize>1?

**Comment by dk-liang** [comment]
"how to actually 'remove' those stopped tokens in the inference stage to reduce inference time when batchsize>1?"

I have the same question. 

Could the authors give some explanation?

**Comment by hongxuyin** [comment]
Hi Mandy, thanks for letting us know. This set yields the accuracy of the provide checkpoint. Can you share your training environment and the exact code you run? Also hi dk, we will update repository with more snippets in coming versions. Stay tuned.
